### PR TITLE
Order Total now in checkout bar #110

### PR DESCRIPTION
**Pull request by matthall8**
Notes: Do not merge before #112 

Updates

- Checkout total is now dynamic based on the basic #110

### CODE REVIEW

File: client/src/pages/BasketPage/BasketPage.js [hunk 1]
```diff
@@ -2,10 +2,20 @@ import React, { Component } from "react";
 import { connect } from "react-redux";
 import { Link } from "react-router-dom";
 import removeBasketItem from "../../actions/removeBasketItem";
-import { MealCard } from "../../components";
+import { MealCard, MoneyButton } from "../../components";
 import { MealName, Delivery, OrderInfo, Cost } from "./BasketPage.style";
 
 class BasketPage extends Component {
+  orderSum = () => {
+    const basket = this.props.basket;
```
Comment (DenisKent) [inline client/src/pages/BasketPage/BasketPage.js]: destructure?

File: client/src/components/MoneyButton/MoneyButton.js [hunk 2]
```diff
@@ -1,24 +1,47 @@
...
+              onClick={() =>
+                this.props.addToBasket(this.props.mealId, this.props.mealInfo)
+              }
+            >
+              Add to Basket item
+            </AddToBasket>
+          </Order>
+        ) : (
+          <Order>
+            <Price>Total: £{this.props.total}</Price>
```
Comment (DenisKent) [inline client/src/components/MoneyButton/MoneyButton.js]: var discount = (price / listprice).toFixed(2);
example of how to fix 2 dp

File: client/src/components/MoneyButton/MoneyButton.js [hunk 3]
```diff
@@ -1,24 +1,48 @@
...
-  <MoneyButtonWrapper>
-    <Order>
-      <Price>Price: £{props.mealInfo.price}.</Price>
-      <button onClick={() => props.addToBasket(props.mealId, props.mealInfo)}>
-        Add to Basket item
-      </button>
-    </Order>
-  </MoneyButtonWrapper>
-);
+class MoneyButton extends Component {
```
Comment (jema28) [inline client/src/components/MoneyButton/MoneyButton.js]: Why did you change this back to a class?
